docs/ember/component-signatures.md

```diff
@@ -1,18 +1,30 @@
-While Glimmer components accept `Args` as a type parameter, and Ember components accept no type parameters at all, the Glint version of each accepts `Signature`, which contains types for `Element`, `Args` and `Yields`.
+Since the implementation of [RFC 748], Glimmer and Ember components accept a `Signature` type parameter as part of their
```

Comment:
Do we care about the history? From a teaching POV, I think it’s more or less irrelevant now?

```suggestion
Glimmer and Ember components accept a `Signature` type parameter as part of their
```

Reply:
My thinking in including this was that for many folks who don't necessarily follow the RFCs repo as closely as you or I do, this may be their first exposure to signatures as a concept. Nearly any existing documentation and code examples they'll see in the wild today will only show `Component<Args>`, including in many cases their own experience writing TS components in the past.

By calling out the RFC first thing on this page, my hope was to (a) reassure readers that this typing scheme is "just Ember" rather than something special Glint is inventing, and (b) provide easy access to the context behind how we got to this design from `Args` for those who may want to better understand it.

As TS is incorporated into the Ember docs themselves with guidance around signatures, and as code in the wild begins to be written this way, I agree that the history won't bring anything here from a teaching perspective. Or do you think it's already just a distraction today?

Reply:
Ah, this is a good point. I have a serious Curse of Knowledge™ going on, I think—your reasoning here makes good sense. Let's leave it in!